src/backend/core/api/viewsets.py

```diff
@@ -203,7 +203,7 @@ def get_queryset(self):
                 Prefetch(
                     "identities",
                     queryset=models.Identity.objects.filter(is_main=True),
-                    to_attr="main_identity",
+                    to_attr="_identities_main",
```

Comment:
why change ?

Reply:
the problem with `main_identity` is that you make people believe that it is an identity... and then you have to write `user.main_identity[0]` and it gets weird. In fact this attribute is a subquery so `identities_main` best conveys the fact that there are many identities but that I'm interested in the main one here...